mindjunkies/lecture/management/commands/populate_lecture.py

```
@@ -0,0 +1,86 @@
+                    for user in visit_users:
+                        LastVisitedModule.objects.get_or_create(
+                            user=user,
+                            module=module,
+                            lecture=lecture,
+                            defaults={
+                                'last_visited': datetime.now() - timedelta(days=random.randint(0, 30))
+                            }
+                        )
+                        self.stdout.write(f"Created last visited entry for {user.username} on {lecture.title}")
+
+        self.stdout.write(self.style.SUCCESS("Lecture data population completed successfully!"))
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)